git/init.go

```diff
@@ -10,8 +11,7 @@ type InitOptions struct {
 	Quiet bool
 	Bare  bool
 
-	// Not implemented
-	TemplateDir File
+	Template string
```

Comment:
The variable rename doesn't bother me, but why change it from File to string? File is a type of string which adds some helper methods for interacting with files. You can always cast it back to a string to interact with the Go standard library, but you lose the compile time benefits of type checking if you don't type it as a File and someone accidentally passes another kind of string.

You can always use `flags.String` and then cast instead of using `flags.StringVar` in the cmd package.